src/renderer/src/routes/pages/Login/index.tsx

```diff
@@ -38,7 +37,7 @@ export const LoginPage = () => {
   const handleSubmit = async (data: TFormData) => {
     try {
       await login(data.password)
-      navigate(location.state.from ?? '/')
+      navigate('/portfolio')
```

Comment:
Why do you remove location.state.from ?

```suggestion
      navigate(location.state.from ?? '/portfolio')
```